services/addons/images/iss_health_check/iss_health_checker.py

```diff
@@ -1,118 +1,124 @@
-                rsu_data[enrollment_status["_id"]][
-                    "provisionerCompany"
-                ] = enrollment_status["provisionerCompany_id"]
-                rsu_data[enrollment_status["_id"]]["entityType"] = enrollment_status[
-                    "entityType"
-                ]
-                rsu_data[enrollment_status["_id"]]["project_id"] = enrollment_status[
-                    "project_id"
-                ]
-                rsu_data[enrollment_status["_id"]]["deviceHealth"] = enrollment_status[
-                    "deviceHealth"
-                ]
-
-                # If the device has yet to download its first set of certs, set the expiration time to when it was enrolled
-                if "authorizationCertInfo" in enrollment_status["enrollments"][0]:
-                    rsu_data[enrollment_status["_id"]][
-                        "expiration"
-                    ] = enrollment_status["enrollments"][0]["authorizationCertInfo"][
-                        "expireTimeOfLatestDownloadedCert"
-                    ]
-                else:
-                    rsu_data[enrollment_status["_id"]]["expiration"] = None
-
-            messages_processed = messages_processed + 1
-
-        page = page + 1
-
-    logging.info("Processed {} messages".format(messages_processed))
-    return rsu_data
-
-
-def insert_scms_data(data):
-    logging.info("Inserting SCMS data into PostgreSQL...")
-    now_ts = datetime.strftime(datetime.now(), "%Y-%m-%dT%H:%M:%S.000Z")
-
-    query = (
-        'INSERT INTO public.scms_health("timestamp", health, expiration, rsu_id) VALUES'
-    )
-    for value in data.values():
-        health = "1" if value["deviceHealth"] == "Healthy" else "0"
-        if value["expiration"]:
-            query = (
-                query
-                + f" ('{now_ts}', '{health}', '{value['expiration']}', {value['rsu_id']}),"
-            )
-        else:
-            query = query + f" ('{now_ts}', '{health}', NULL, {value['rsu_id']}),"
-
-    pgquery.write_db(query[:-1])
-    logging.info(
-        "SCMS data inserted {} messages into PostgreSQL...".format(len(data.values()))
-    )
-
-
-if __name__ == "__main__":
-    # Configure logging based on ENV var or use default if not set
-    log_level = (
-        "INFO" if "LOGGING_LEVEL" not in os.environ else os.environ["LOGGING_LEVEL"]
-    )
-    logging.basicConfig(format="%(levelname)s:%(message)s", level=log_level)
-
-    scms_statuses = get_scms_status_data()
-    insert_scms_data(scms_statuses)
+from datetime import datetime
+import requests
+import logging
+import os
+import iss_token
+import common.pgquery as pgquery
+
+
+def get_rsu_data():
+    result = {}
+    query = (
+        "SELECT jsonb_build_object('rsu_id', rsu_id, 'iss_scms_id', iss_scms_id) "
+        "FROM public.rsus "
+        "WHERE iss_scms_id IS NOT NULL "
+        "ORDER BY rsu_id"
+    )
+    data = pgquery.query_db(query)
+
+    logging.debug("Parsing results...")

```

Comment:
Can we create and use logger for this module? logger = logging.getLogger(__name__)

Reply:
A module-specific logger is now being used for the ISS Health Check addon.